src/shell/shell.cpp

```diff
@@ -647,6 +647,8 @@ void SHELL_Init() {
 	MSG_Add("SHELL_FILE_NOT_FOUND", "File not found - '%s'\n");
 	MSG_Add("SHELL_FILE_EXISTS", "File '%s' already exists.\n");
 	MSG_Add("SHELL_DIRECTORY_NOT_FOUND", "Directory not found - '%s'\n");
+	MSG_Add("SHELL_READ_ERROR", "Error reading from file - %s\n");
```

Comment:
1. Change `%s` to `'%s'`, like in the strings above.
2. Wouldn't it sound better to write _Error reading file_ instead of _Error reading from file_, and similarly _Error writing file_?

Reply:
Agree on both points.

Reply:
Fixed